image_proc/src/nodelets/crop_decimate.cpp

```diff
@@ -195,8 +195,24 @@ void CropDecimateNodelet::imageCb(const sensor_msgs::ImageConstPtr& image_msg,
     config.height &= ~0x1;    
   }
 
+  // TODO(lucasw) need to incorporate the x_offset and y_offset within info_msg
+
+  // TODO(lucasw) should generate a black image where there isn't any source image,
+  // could be a dynamic reconfigure option
```

Comment:
Please remove these comments and create issues on this repo associated with these.

Reply:
Done, #428 and #429